opencv_example.c: In function 'Filter':
opencv_example.c:194:45: error: too few arguments to function 'cvHaarDetectObjects'
/usr/include/opencv2/objdetect/objdetect.hpp:137:15: note: declared here

--- origsrc/vlc-1.1.11/modules/video_filter//opencv_example.c	2010-04-12 19:22:27.000000000 -0500
+++ src/vlc-1.1.11/modules/video_filter//opencv_example.c	2011-08-11 17:12:15.205827300 -0500
@@ -191,7 +191,7 @@ static picture_t *Filter( filter_t *p_fi
         CvSeq *faces = cvHaarDetectObjects( p_img[0], p_sys->p_cascade,
                                             p_sys->p_storage, 1.15, 5,
                                             CV_HAAR_DO_CANNY_PRUNING,
-                                            cvSize(20, 20) );
+                                            cvSize(0, 0), cvSize(20, 20) );
         //create the video_filter_region_info_t struct
         if (faces && (faces->total > 0))
         {
